src/genericComponents/Dropdown.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Col, Form } from 'react-bootstrap';
+
+const DropdownMenu = (props: any) => {
+  const { onChange, values } = props;
+  //do mapping that returns options
+  return (
+    <Form.Group as={Col} controlId="formGridState">
+      <Form.Label>Select</Form.Label>
+      <Form.Control as="select" defaultValue="Choose..." onChange={onChange}>
+        {values.map((value: string) => {
```

Comment:
Do:
```
{values.map((value: string) => <option value={value}>{value}</option>)}
```

We'll also need to add a prop for selected value, and then do some logic here to know if the option is the one selected. For example:

```
{values.map((value: string) => <option value={value} selected={value === selectedValue}>{value}</option>)}
```